client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -1,268 +1,94 @@
 @import 'assets/stylesheets/gutenberg-base-styles';
 @import '~@wordpress/base-styles/colors';
-
-$design-selector-grid-gap: 4.5em;
-
-$design-selector-max-width: 1100px;
-$design-selector-selection-space: 285px + $header-height;
-$page-selector-width-offset: 20px;
-$page-selector-shadow-size: 12px;
-
-.design-selector__title {
-	color: $dark-gray-800;
-	font-size: 2em;
-	font-weight: bold;
-	margin-bottom: 0.25em;
-	text-align: center;
-}
-
-.design-selector__subtitle {
-	color: $dark-gray-800;
-	font-size: 1.3em;
-	text-align: center;
-}
-
-.design-selector__grid-container {
-	position: absolute;
-	padding-bottom: 48px;
-	padding-top: 19px;
-
-	&.is-page-selector-open {
-		overflow-y: hidden;
-		height: 100vh;
-	}
-}
-
-.design-selector__grid {
-	display: grid;
-	grid-gap: $design-selector-grid-gap;
-
-	@include breakpoint( '>660px' ) {
-		grid-template-columns: 1fr 1fr;
-	}
-}
-
-.design-selector__page-layout-backdrop {
+@import '../../mixins';
+
+@font-face {
+	font-display: swap;
+	font-family: 'Recoleta';
+	font-weight: 400;
+	src: url( 'https://s1.wp.com/i/fonts/recoleta/400.eot' );
+	src: url( 'https://s1.wp.com/i/fonts/recoleta/400.eot?#iefix' ) format( 'embedded-opentype' ),
+		url( 'https://s1.wp.com/i/fonts/recoleta/400.woff2' ) format( 'woff2' ),
+		url( 'https://s1.wp.com/i/fonts/recoleta/400.woff' ) format( 'woff' ),
+		url( 'https://s1.wp.com/i/fonts/recoleta/400.ttf' ) format( 'truetype' );
+}
+
+.design-selector {
+	width: 90%;
+	height: 90%;
+	background-color: var( --contrastColor );
+	color: var( --mainColor );
```

Comment:
Just occured to me that these are pretty generic names — no need I'm this PR but we should prolly prefix them to avoid issues with calypso components or some day in WP admin.

Good for a follow-up PR.

Reply:
https://github.com/Automattic/wp-calypso/issues/40085